lib/stat_tracker.rb

```diff
@@ -188,7 +188,22 @@ def rival(team_id)
   end
 
   def best_offense
-    @teams[0][:name]
+    # hash to store {team_id => avg goals/game}
+    team_id_goals_hash = Hash.new { |h, k| h[k] = [] }
+    # turn CSV::Rows => Hashes
```

Comment:
i like this idea.. maybe we can eventually turn rows into their own class instances